### ISSUE TITLE
about the experiment in IMDN

### ISSUE THREAD

**Issue by yuqiangchu** [Zheng222/IMDN#19]
Hi,Zheng!
I trained the IMDN in a regular manner with the lr 2e-4.The lr reduced by half per 200 epochs for 1000 epochs totally.However,when I tested the network ,the results i got were lower than yours.After that, I increased the epochs to 300 when the lr is 2e-4 and 1e-4 and the total epochs was increased in 1200. But finally, the results were still lower than yours.The psnr and ssim i got were obtained by matlab in y channel.The experiment results i tested are as followed :
para|MAds|        Set5       |        Set14      |        B100      |    Urban100   |    manga109   |
715 | 40.9 | 32.14/0.8942 | 28.53/0.7806 | 27.53/0.7351 | 26.00/0.7828 | 30.37/0.9067
So, could you tell me what i should do to further improve the final results?

**Comment by Zheng222** [comment]
@yuqiangchu You need train x2 model firstly and then using the pretrained ckpt to initialize the x4 model.

**Comment by yuqiangchu** [comment]
@Zheng222 But at the x2 scale, the results are still lower than yours as showed in the paper.The results I got at x2 scale are , Set 5 37.99/0.9605, Set14 33.49/0.9170, B100 32.13/0.8991, Urban 100 32.01/0.9269.
So should the model at the x2 scale be trained twice by using the the pretrained x2 ckpt?

**Comment by yuqiangchu** [comment]
@Zheng222  As I used the pretrained x2 ckpt to retrain the x2 model, the results slightly increased but still lower than yours. Should the model be trained several times untill the results become steady?

**Comment by Zheng222** [comment]
@yuqiangchu You can train the x2 model serveral times when the learning rate is 2e-4 or 1e-4.

**Comment by kk6398** [comment]
@yuqiangchu  hello， I want to ask you if you deal with this question?

**Comment by liang-zhenhuan** [comment]
> @Zheng222 But at the x2 scale, the results are still lower than yours as showed in the paper.The results I got at x2 scale are , Set 5 37.99/0.9605, Set14 33.49/0.9170, B100 32.13/0.8991, Urban 100 32.01/0.9269. So should the model at the x2 scale be trained twice by using the the pretrained x2 ckpt?

您好，我遇到了和您一样的问题，请问您解决了吗？可否加我QQ：2481654160向您请教一下，谢谢！
